cypress/components/popover-container/popover-container.cy.tsx

```diff
@@ -280,13 +280,33 @@ context("Test for Popover Container component", () => {
       }
     );
 
+    it("should close Popover Container using escape keyboard key", () => {
+      CypressMountWithProviders(
+        <PopoverContainer title="Cypress is awesome">Contents</PopoverContainer>
+      );
+
+      popoverSettingsIcon().click();
+      popoverCloseIcon().trigger("keydown", keyCode("Esc"));
```

Comment:
```suggestion
      pressESCKey();
```

**suggestion**: I think this is the better test here, and not just because it's less code 😉 

You're specifically triggering `keydown` on the close icon, but (inlike pressing return with the close icon focused) which element the event is triggered on doesn't matter, and in the component you've (correctly imo) added the `keydown` handler to the whole `document`. So we should test that Escape works from anywhere on the page, not just when the close icon is focused.

Reply:
Thank you. I completely forgot that `pressESCKey()` even existed. I'll get that changed.